docs/guides/migrate-from-buidler.md

```diff
@@ -6,7 +6,7 @@ This guide will teach you how to migrate your project from Buidler into Hardhat.
 
 ## Installing the Hardhat packages
 
-The first thing you need to do to migrate your project, is installing the new npm packages.
+The first thing you need to do to migrate your project is to install the new npm packages.
```

Comment:
When should we remove this section? @fvictorio @fzeoli

Reply:
I think we could delist it already, and if someone asks through support channels we can share it

Reply:
De-listed in https://github.com/nomiclabs/hardhat/pull/1554/commits/967e92181fd3bdcecc90b840b0193886bae829af